README.md

```diff
@@ -7,3 +7,35 @@ slog is a library for capturing structured log information. In contrast to "trad
 * captures arbitrary key-value metadata on each log event
 
 slog forwards messages to [`log`](https://golang.org/pkg/log/) by default. But you probably want to write a a custom output to make use of the context and metadata. At [Monzo](https://monzo.com/), slog captures events both on a per-service and a per-request basis (using the context information) and sends them to a centralised logging system. This lets us view all the logs for a given request across all the micro-services it touches.
+
+## Usage
+
+Internally at Monzo, we recommend that users always prefer structured logging where possible. An example of using slog for this would be:
```

Comment:
This isn't formalised yet but will be the recommendation from backend-platform soon.